epfl-menus.php

```diff
@@ -1558,13 +1559,40 @@ static private function _get_publish_controller ($menu) {
      * Shall be called whenever $menu changes (from the point
      * of view of @link get_menu)
      */
-    static function menu_changed ($menu, $causality = NULL) {
-        $publisher = static::_get_publish_controller($menu);
+    static function menu_changed ($menu) {
+        $site_url = trim(get_site_url());
+        $site_url = trailingslashit( $site_url );
```

Comment:
Please don't use spaces around `$site_url` here.

Please use `untrailingslashit` and remove the next `if` clause.